integration/js/server.js

```diff
@@ -33,13 +33,19 @@ createMeeting = async (baseUrl) => {
   }
   meetingCreationStatus = "creating";
   meetingTitle = uuidv4();
-  const createMeetingUrl = `${baseUrl}join?title=${meetingTitle}&name=MeetingOwner&region=us-east-1`;
+  let url = `${baseUrl}`;
+  let index = url.indexOf('?');
```

Comment:
url.includes('?')

Reply:
Can we use this instead?

Reply:
I needed the index for the `substring` so used `indexOf`.